plugins/CMakeLists.txt

```diff
@@ -6,9 +6,9 @@ if(NOT TOP_CMAKE_WAS_SOURCED)
 endif()
 
 
-#if(EXISTS "${CMAKE_SOURCE_DIR}/plugins/cdvdGigaherz" AND cdvdGigaherz)
-#	add_subdirectory(cdvdGigaherz)
-#endif()
+if(EXISTS "${CMAKE_SOURCE_DIR}/plugins/cdvdGigaherz" AND cdvdGigaherz)
+	add_subdirectory(cdvdGigaherz/src)
```

Comment:
Oh, I thought you were initially talking about tabs/spaces and not the random "tab indentation sometimes takes a vacation on Github" thing. I've seen that a few times myself.